Supertext.Sdl.Trados.FileType.JsonFile/JsonFileWriter.cs

```diff
@@ -89,7 +90,20 @@ public void ProcessParagraphUnit(IParagraphUnit paragraphUnit)
                 return;
             }
 
-            var targetText = _segmentReader.GetText(paragraphUnit.Target);
+            var sb = new StringBuilder();
```

Comment:
Wouldn't it make more sense to have this functionality in the segment reader and provide a GetTargetText(SegmentPairs) method?